scripts/start-op-geth.sh

```diff
@@ -58,7 +58,6 @@ exec geth \
   --rollup.disabletxpoolgossip=true \
   --port="${PORT__OP_GETH_P2P:-39393}" \
   --discovery.port="${PORT__OP_GETH_P2P:-39393}" \
-  --db.engine=pebble \
```

Comment:
Why do we need to remove this? Does it fail to load datadirs with `leveldb` if this is set?

Anyways, the flag default to `pepple`, so behaviour for new nodes stays the same.

Reply:
That is exactly it, if explicitly set then only pebble db engines can be used. If unset then it will first use the db engine found in the datdir, and if there is no datadir it will default to pebble.